core/dbt/events/base_types.py

```diff
@@ -88,6 +90,21 @@ def get_pid(self) -> int:
             self.pid = os.getpid()
         return self.pid
 
+    def get_node_info(self) -> Any:
+        if self.report_node_data:
+            return {
+                "type": 'node_status',
```

Comment:
Not convinced we need this — I can't think of any events right now that should have node info, and aren't `'node_status'` in one way or another

Reply:
Realized this is here for functional/plumbing reasons, not user-facing context